packages/richtext-lexical/src/index.ts

```diff
@@ -83,41 +85,28 @@ export function lexicalEditor(args?: LexicalEditorProps): LexicalRichTextAdapter
       resolvedFeatureMap = result.resolvedFeatureMap
     }
 
-    const featureI18n = finalSanitizedEditorConfig.features.i18n
-    for (const lang in i18n) {
-      if (!featureI18n[lang as keyof typeof featureI18n]) {
-        featureI18n[lang as keyof typeof featureI18n] = {
-          lexical: {},
-        }
-      }
-      // @ts-expect-error - vestiges of when tsconfig was not strict. Feel free to improve
-      featureI18n[lang].lexical.general = i18n[lang]
+    const featureI18n: Partial<GenericLanguages> = finalSanitizedEditorConfig.features.i18n
+    for (const _lang in i18n) {
+      const lang = _lang as keyof typeof i18n
+      const lexicalI18nForLang = ((featureI18n[lang] ??= {}).lexical ??=
+        {}) as GenericTranslationsObject
+
+      lexicalI18nForLang.general = i18n[lang] ?? {}
     }
 
     config.i18n.translations = deepMergeSimple(config.i18n.translations, featureI18n)
 
     return {
-      CellComponent: {
-        path: '@payloadcms/richtext-lexical/rsc#RscEntryLexicalCell',
-        serverProps: {
-          admin: args?.admin,
-          sanitizedEditorConfig: finalSanitizedEditorConfig,
-        },
-      },
-      DiffComponent: {
-        path: '@payloadcms/richtext-lexical/rsc#LexicalDiffComponent',
-        serverProps: {
-          admin: args?.admin,
-          sanitizedEditorConfig: finalSanitizedEditorConfig,
-        },
-      },
+      CellComponent: '@payloadcms/richtext-lexical/rsc#RscEntryLexicalCell',
```

Comment:
Removed passing severProps that were completely unused. They just unnecessarily blew up the size of this field config.